### PR TITLE
added the branch, first push I didn't check out the branch before i pushed it.

### PR DESCRIPTION
**Pull request by omarbowen01**
…rting work

### CODE REVIEW

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
This PR has no code to review... Please update this PR with the code to review Iand I will get around to it.

**Comment by omarbowen01** [comment]
PR has been updated with code. Please let me know if you need anything else. Thank You!

File: code-challenges.test.js [hunk 1]
```diff
@@ -33,27 +33,27 @@
...
+const arrayCap = (arr) => {
 
+  let mappedArr = arr.map(obj)=>
 
+  let splitName = obj.name.split(" ")
+  let mappedCapitalizedName = splitName.map(string=> string[0].toUpperCase() + string.slice(1))
+    return `${mappedCapitalizedName.join(" ")} is ${obj.occupation}.`
+})
+    return mappedArr
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:56]: nice!

File: code-challenges.test.js [hunk 2]
```diff
@@ -73,22 +73,20 @@
...
+// return typeof v === "number"
+// return filtered.map with value and v modulo 3
+
+const remainders = (arr) =>
+{
+  let filtered = arr.filter(v) => {
+      return typeof v === "number"
+})
+      return filtered.map((v) => v % 3)
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:88]: Very clean!

File: code-challenges.test.js [hunk 3]
```diff
@@ -106,15 +104,16 @@
...
+
+//create a function named numbersCubed that returns an array of only the remainders of numbers when divided by 3
+// use arr as a parameter
+// create variable of sum eqaul to zero
+// use .filter to iterate through array to return the sum of the reminders divided by three
+// return new sum
+
+const numbersCubed = (arr) => {
+  let sum =0
+  arr.filter(v=> sum += v ** 3)
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:116]: You used filter effectively here. Recommend looking into .reduce().

File: rails-commenting.rb [hunk 4]
```diff
@@ -20,14 +20,14 @@ def show
...
-  # shows a new form with the data from BlogPost.new
+  #Method named new is defined with an instance variable called @post. The responsibility of new is to display a text form for users to input data.
   def new
     @post = BlogPost.new
   end
 
   def create
     # ---5)
-    # post new data for the blog post
+    #@post is an instance variable called on the Database and using the create method to make a new instance in the database using the private method blog_post_params.
```
Comment (WalkerTexas-coder) [inline rails-commenting.rb:30]: Well done!

File: interview-questions.md [hunk 5]
```diff
@@ -50,7 +51,7 @@ action: "GET"    location: /students/2/edit
 action: "PATCH"  location: /students/2      
 - update the 2nd student
 action: "DELETE" location: /students/2      
-- delete the 2nd student
+- delete the 2nd student.
 
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
```
Comment (WalkerTexas-coder) [inline interview-questions.md:57]: This is almost correct. What we want to describe is what the user will experience on our platform so we can build out the features that will match that experience. 

"As a user I can see a plus button that will navigate me to a page where I can create a to-do item"

**Review by WalkerTexas-coder — APPROVED** [review]
Okay Omar this sis good stuff. Thank you for figuring out some of the git issues but I do want to see that you know how to research and get these issues squared away. Good work on the assessment. Double check your user stories and perhaps practice writing some out before you head into interviews so you can ask questions about what the context of the internship workflow will be like. Approved!

**Pull request merged**